src/glang/lib/std/string.c3

```diff
@@ -154,3 +225,13 @@ function[Int] unstringify : (view : StringView) -> TypeIf<Int, IsIntegral<Int>>
 
     return result;
 }
+
+function strlen : (cstr : u8[&]) -> isize = {
```

Comment:
So we already have that function, but it's called `length`.
```rust
function length : (str: CString&) -> isize = {
    return str.data:strlen();
}
```

You could argue that we should avoid the C library conventions...
In the future I'd like to replace `CString` with `CStringView` entirely.

Reply:
Well, this would just become:
```
function length : (str: CString&) -> isize = {
    return str:strlen();
}
```

And strlen would still be correctly restricted to cstrings